pkg/hintrunner/zero/hintcode.go

```diff
@@ -84,7 +84,7 @@ const (
 	// ------ Dictionaries hints related code ------
 
 	// ------ Other hints related code ------
-	allocSegmentCode string = "memory[ap] = segments.add()"
-	vmEnterScopeCode string = "vm_enter_scope()"
-	vmExitScopeCode  string = "vm_exit_scope()"
+	allocSegmentCode     string = "memory[ap] = segments.add()"
+	memcpyEnterScopeCode string = "vm_enter_scope({'n': ids.len})"
```

Comment:
Oh I see you wrote on top of `vmEnterScopeCode`. These two are different hints so you need to handle them separately

Reply:
Got it, should be fixed now!